src/main/java/org/sonar/plugins/github/MarkDownUtils.java

```diff
@@ -35,7 +35,7 @@
 @InstantiationStrategy(InstantiationStrategy.PER_BATCH)
 public class MarkDownUtils {
 
-  private static final String IMAGES_ROOT_URL = "https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/";
+  private static final String IMAGES_ROOT_URL = "http://sonarsource.github.io/sonar-github/";
```

Comment:
Awesome. I am not 💯 sure, but you can have Github on-premise (https://enterprise.github.com/features#features), it might be that in those cases there is a possibility that gh's image proxy (https://help.github.com/articles/about-anonymized-image-urls/) is not enabled or can be disabled. In those cases using a http url would cause mixed-content issues.